akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -764,6 +765,13 @@ trait DefaultSSLContextCreation {
   def createDefaultClientHttpsContext(): HttpsConnectionContext =
     createClientHttpsContext(sslConfig)
 
+  // currently the same configuration as client by default, however we should tune this for server-side apropriately (!)
+  def createServerHttpsContext(sslConfig: AkkaSSLConfig): HttpsConnectionContext = {
+    log.warning("Automatic server-side configuration is not supported yet, will attempt to use client-side settings. " +
```

Comment:
Instead as part of this PR I'll start doing the "simple server side using ssl-config".